### PR TITLE
update customizer featured color to include buy now & rsvp button

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/71093

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/6ad931e4a6e375f8074170a20b98e80c9546c6fc/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:30](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-event.php#L30)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-event.php#L58)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:64](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-event.php#L64)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/0fe42588f990689c58bb06b809b1c7fbbd5c3b24/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Customizer/General_Theme.php [hunk 1]
```diff
@@ -115,6 +116,18 @@ public function get_css_template( $template ) {
...
+				.tribe-button {
+					background:' . $background_color . ' -webkit-linear-gradient( top, ' . $background_color . ', #' . $button_gradient . ');
+					background:' . $background_color . ' linear-gradient( to bottom, ' . $background_color . ', #' . $button_gradient . ');
+					}
+					
+				.tribe-button:hover,
+				.tribe-button:active,
+				.tribe-button:focus {
+					background-color: ' . $background_color . ';
+					background-color: ' . $background_color . ';
```
Comment (borkweb) [inline src/Tribe/Customizer/General_Theme.php]: Looks like an accidental duplication here

File: src/views/day/single-event.php [hunk 2]
```diff
@@ -25,6 +25,10 @@
 <?php if ( tribe_get_cost() ) : ?>
 	<div class="tribe-events-event-cost">
 		<span><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php
+			/** This action is documented in the-events-calendar/src/views/list/single-event.php */
+			do_action( 'tribe_events_inside_cost' )
```
Comment (borkweb) [inline src/views/day/single-event.php]: Untab this line and the previous by one

File: src/views/list/single-event.php [hunk 3]
```diff
@@ -24,56 +24,64 @@
...
+			 *
+			 * @since 4.5
+			 */
+			do_action( 'tribe_events_inside_cost' )
+		?>
 	</div>
 <?php endif; ?>
 
-<!-- Event Title -->
+	<!-- Event Title -->
```
Comment (borkweb) [inline src/views/list/single-event.php]: Undo all of these added tabs ◀️

File: src/views/list/single-featured.php [hunk 4]
```diff
@@ -68,6 +61,17 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+	<!-- Event Cost -->
+<?php if ( tribe_get_cost() ) : ?>
+	<div class="tribe-events-event-cost featured-event">
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php
+			/** This action is documented in the-events-calendar/src/views/list/single-event.php */
+			do_action( 'tribe_events_inside_cost' )
```
Comment (borkweb) [inline src/views/list/single-featured.php]: Untab this and the previous line

File: src/views/list/single-featured.php [hunk 5]
```diff
@@ -68,6 +61,17 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+	<!-- Event Cost -->
```
Comment (borkweb) [inline src/views/list/single-featured.php]: Untab this line

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:29](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-event.php#L29)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:57](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-event.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:63](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-event.php#L63)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/add42123c0142c81c798454c716666ed6165af37/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:29](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-event.php#L29)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:57](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-event.php#L57)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:63](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-event.php#L63)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:67](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-event.php#L67)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-event.php#L83)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-featured.php:67](https://github.com/moderntribe/the-events-calendar/blob/d1a7e743a4607f098bc194b30fbeeba718494884/src/views/list/single-featured.php#L67)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$events_label_plural' | [single-event.php:28](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L28)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:37](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L37)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:39](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L39)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$events_label_singular' | [single-event.php:50](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L50)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:62](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L62)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$events_label_singular' | [single-event.php:83](https://github.com/moderntribe/the-events-calendar/blob/bb0d028ef8f9d4291f8be24688ce6e3afa6804e5/src/views/list/single-event.php#L83)

> via jenkins/codesniffer-phpcs

**Pull request merged**